k8s/007-projects/001-custom/all-ingress.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
 spec:
   rules:
   # - host: aks-helloworld.<CLUSTER_SPECIFIC_DNS_ZONE>
-  - host: apps.8e1005bc066142c3b353.southafricanorth.aksapp.io
+  - host: apps.d8db806e5a8b43dca905.southafricanorth.aksapp.io
```

Comment:
Why the host name to change? Is this in sync with our demo ??

Reply:
I think i have changed it to mine when i ran ingress should i change it to previous?

Reply:
Yes pls change it back

Reply:
Done changes as requested